examples/jsm/math/MeshSurfaceSampler.js

```diff
@@ -156,7 +179,13 @@ class MeshSurfaceSampler {
 
 	}
 
-	sampleFace( faceIndex, targetPosition, targetNormal, targetColor ) {
+	emod( n, m ) {
```

Comment:
Please make this a separate function rather than a class method.

Reply:
not sure where you want the function